src/lastlog.c

```diff
@@ -88,7 +88,7 @@ static void print_one (/*@null@*/const struct passwd *pw)
 	char ptime[80];
 
 #ifdef HAVE_LL_HOST
-	int maxIPv6Addrlen;
+	int maxIPv6Addrlen = 0;
```

Comment:
Would it be better to just define a (build-time-computed) const at the top of the function?

Reply:
Do you mean the following?
```
#ifdef HAVE_LL_HOST
	const int maxIPv6Addrlen = 0;
#endif
```

Reply:
No, I meant

```
#ifdef HAVE_LL_HOST
	const int maxIPv6Addrlen = 25+1+IFNAMSIZ;
#endif
```